libcloud/compute/drivers/ecs.py

```diff
@@ -1289,6 +1289,102 @@ def create_public_ip(self, instance_id):
             return findtext(resp.object, 'IpAddress',
                             namespace=self.namespace)
 
+    def ex_security_group_ingress_permission(self, group_id,
+                                             ip_protocol, portrange,
```

Comment:
Minor consistency thing, please use `port_range` or perhaps `from_port` and `to_port` if you are going to make it look more like the methods in EC2 and CloudStack drivers.